lib/bootevent.js

```diff
@@ -1,38 +0,0 @@
-
```

Comment:
Because it is not necessary anymore at this moment. As we see in the code, we observe two acquire blocks. Which are processed asynchronously as we read from the (documentation)[https://www.npmjs.com/package/bootevent]. But it is obvious that the second block is dependent on the first block as we use the `ctx.store` in the second block, which is initialized in the first block. So we have to wait for second block to execute as the first block is finished (and the `ctx.store` variable is initialized.

The problem is observable in the build logs as the initialization method of the [Singleton-like](http://stackoverflow.com/questions/137975/what-is-so-bad-about-singletons) pattern, as we see four times the `Connecting to MongoDB` [line](https://travis-ci.org/nightscout/cgm-remote-monitor/jobs/67610160#L437). This will run just right as the variable within the store class just gets overwritten a couple of times, but it is smelly in terms of code quality.
We do not observe this in the [build-log](https://travis-ci.org/nightscout/cgm-remote-monitor/jobs/67378958#L426) of pull-request.

When we wait for the store-callback, we reduce the bootevent to a single acquire block, which makes the boot-event a bit redundant. Therefore I decided to remove it.